web/annotation_editor_layer_builder.css

```diff
@@ -18,11 +18,15 @@
   --outline-color: #0060df;
   --outline-around-width: 1px;
   --outline-around-color: #f0f0f4;
+  --hover-outline-around-color: var(--outline-around-color);
   --focus-outline: solid var(--outline-width) var(--outline-color);
   --unfocus-outline: solid var(--outline-width) transparent;
   --focus-outline-around: solid var(--outline-around-width)
     var(--outline-around-color);
-  --hover-outline: dashed var(--outline-width) var(--outline-color);
+  --hover-outline-color: #0250bb;
+  --hover-outline: solid var(--outline-width) var(--hover-outline-color);
+  --hover-outline-around: solid var(--outline-around-width)
+    var(--hover-outline-around-color);
```

Comment:
With these changes it's no longer possible to *as easily* tell the selected vs hovered editors apart, which feels like it'll make selecting multiple editors slightly more difficult than before.

This is *particularly noticeable* with FreeText-editors, since those don't have resizer-elements, and selected/hovered FreeText-editors now look *identical*.

Reply:
The same as above: I'll raise it, but we'll have to do with this design for the moment.

Reply:
I really cannot help feeling that the specification was created by looking only at the Stamp-editor, and completely missing that these changes also affect e.g. the FreeText-editor as well :-(

Reply:
@Snuffleupagus I think that is a good point you've raised. 

We will update the color of the central line in the hover state to be a grey color available in our acorn design system. #8F8F9D. This will accommodate for a larger visual change while remaining accessible in both dark and light modes.